examples/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -260,15 +273,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 7,
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "WARNING:tensorflow:From ../../reco_utils/recommender/deeprec/models/sequential/sequential_base_model.py:43: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.\n",
```

Comment:
Is removing TF warning change coming in? @miguelgfierro

Reply:
work in progress